backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -242,7 +321,11 @@ public String getPatientTribalAffiliation() {
 
   @JsonProperty("Patient_preferred_language")
   public String getPatientPreferredLanguage() {
-    return patient.map(Person::getPreferredLanguage).orElse("");
+    String defaultLanguage = patient.isPresent() ? patient.get().getPreferredLanguage() : "";
```

Comment:
Got it! Yep that makes sense

Reply:
I saw @MauriceReeves-usds mentioned that RS could add `Other` as an option on their side. If that's happened we should be able to include that in the language map. Right now on this branch (i.e. as of [3e6ce6d](https://github.com/CDCgov/prime-simplereport/pull/3898/commits/3e6ce6d92dacbd9fdecbdc79ccf824f992c4c77b)) I think it will just be sent through as the string `"Other"`.

Reply:
Just to be clear, is the purpose of this ternary to default to an empty string if the preferred language isn't in the language map? If so I don't think it's quite correct - in practice, the default language returned by `orElse` will just be the full language string